### ISSUE TITLE
`yaml-language-server` Schema Support

### ISSUE THREAD

**Issue by Trajamsmith** [krassowski/jupyterlab-lsp#244]
At present, it seems that the [`yaml-language-server`](https://github.com/redhat-developer/yaml-language-server) can only be configured using a `didChangeConfigurationRequest`. Our team would like to be able to validate YAML files against a schema using `jupyterlab-lsp`, but it doesn't seem that there's any way to send the change configuration request from the JupyterLab client.

Is that correct? Is there any plan to support configuration changes using that connection method? 

Thanks!

**Comment by Trajamsmith** [comment]
Would this enhancement allow for the above: https://github.com/krassowski/jupyterlab-lsp/issues/196

**Comment by krassowski** [comment]
Thank you for opening this issue.

While I do not have the bandwidth for it now (probably until June), we are [very welcoming of new contributions](https://github.com/krassowski/jupyterlab-lsp/blob/master/CONTRIBUTING.md) - if you or anyone at the team would like to try to tackle the topic of configuration. 

I think we could start with some simplest working configuration to support your use case rather than try to implement all of the intended configuration features. @bollwyvl might have some more thoughts.

**Comment by bollwyvl** [comment]
Indeed, there are a number of servers that are all but useless without some level of config! Handling config should certainly be the next non-bugfix, non-chore, non-architecture, non-robustness features... though we might have to do some of those. It's a shame I haven't been able to put a bow on #205 yet, as no doubt being able to restart servers easily would be nice for testing it, but here we are.

> Would this enhancement allow for the above: #196

That is correct: we need to add the frontend plumbing to emit the correct LSP messages. I'll do a little updated vignette over there...

I, too, am spread a bit thin right now. As @krassowski says, PRs, _or contribution of any kind_, are very welcome! In this case, if you have (or can fabricate) really quite specific open source use examples that we could try to turn into tests, those would be very welcome on #196, or as links to gists.

**Comment by bollwyvl** [comment]
@trajamsmith how's it been going? do you reckon 0.10.1 works now? I want to do a refresh of all our js dependencies, including language servers, soon, and was hoping we could finally put this one to bed!

**Comment by trajamsmith** [comment]
Hey! Yes, the newest release of the `yaml-language-server` package should work nicely with the extension now. I haven't tested it recently, but I'll make some time to try it out. Hopefully it's as easy as a dependency bump.
